packages/cli/src/commands/deploy/baremetal/SshExecutor.js

```diff
@@ -12,13 +12,16 @@ export class SshExecutor {
   async exec(path, command, args) {
     let sshCommand = command
 
+    const argsString = args?.join(' ')
     if (args) {
-      sshCommand += ` ${args.join(' ')}`
+      sshCommand += ` ${argsString}`
     }
 
     if (this.verbose) {
       console.log(
-        `SshExecutor::exec running command \`${command} ${args.join(' ')}\` in ${path}`,
+        `SshExecutor::exec running command \`${command} ${
+          args ? argsString : ''
```

Comment:
```suggestion
          argsString || ''
```
Would that put it all back on one line again?

Reply:
Yeah if you prefer that then that's fine with me.

Reply:
My vscode linted the file the moment I opened it and put it on more than one line before I made any changes

Reply:
Wouldn't be the first time my VsCode doesn't want to properly format files for me...

Reply:
Could be mine. There's probably 12 different places vscode is picking up linting config at this point 🤷

Reply:
I change it in [38494aa](https://github.com/redwoodjs/redwood/pull/10663/commits/38494aa03abc454e723a710ced33acc1593f1957) so it went on to one